README.md

```diff
@@ -124,16 +124,19 @@ Animated GIFs will be displayed as still images to reduce loading time and the
 impact on performance of a client website.
 
 
-dev installation
+Dev installation
 ----------------
 
 ```
 git clone git@github.com:styladev/widget.git
 cd widget
-npm i
+npm install
 npm run build
+python -m SimpleHTTPServer
```

Comment:
could create a shortcut command like `npm run dev` that runs both build and the python script